app-admin/aerospike-amc-community/aerospike-amc-community-4.0.19-r1.ebuild

```diff
@@ -15,7 +15,7 @@ IUSE=""
 RDEPEND=""
 DEPEND="${RDEPEND}"
 
-S=$WORKDIR
+S=${WORKDIR}
```

Comment:
There's a bunch more of these but I wasn't aiming to fix them because it's one of those more-style things, whereas {} is covered by QA [policy](https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101).

Reply:
I understand.
My reasoning was that `${WORKDIR}` is dependent on [user configuration](https://wiki.gentoo.org/wiki//etc/portage/make.conf#PORTAGE_TMPDIR) and might contain spaces (I know it is bad idea, but a user might do it as it isn't discouraged).

Could it be a good idea to add this to QA policy (specially for WORKDIR) or maybe update the documentation [here](WORKDIR)?